### PR TITLE
Update tls-1.0-and-1.1-deprecation-for-office-365.md

### PR DESCRIPTION
**Pull request by qiufasheng**
Updated current TLS 1.2 enforcement status in WW, and included target date for Gallatin.

### CODE REVIEW

**Comment by WorkShay** [comment]
@***@***.******@***.*** ***@***.***>

Hey folks,

Can you both figure out what’s going on and make sure these PRs aren’t getting assigned to me anymore? I don’t remember the login to by Github anymore (yes, I know) so I can’t remove these anyways.

Best,
Shayna

Pronouns: they/them<https://www.mypronouns.org/they-them#:~:text=A%20person%20who%20goes%20by%20%E2%80%9C%20they%20%E2%80%9D,theirs.%20I%20like%20both%20them%20and%20their%20ideas.%E2%80%9D> || she/her<https://www.mypronouns.org/she-her#:~:text=A%20person%20who%20goes%20by%20%E2%80%9C%20she%20%E2%80%9D,hers.%20I%20like%20both%20her%20and%20her%20ideas.%E2%80%9D>
Name pronunciation: [SHā-nah]

File: microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ appliesto:
 # Disabling TLS 1.0 and 1.1 for Microsoft 365
 
 > [!IMPORTANT]
-> We temporarily halted disablement of TLS 1.0 and 1.1 for commercial customers due to COVID-19. As supply chains have adjusted and certain countries open back up, we restarted the TLS 1.2 enforcement rollout on October 15, 2020. Rollout will continue over the following weeks and months.
+> We have already disabled TLS 1.0 and 1.1 for most of Microsoft 365 services in our world wide environment. Rollout will continue over the following weeks and months.
+For Microsoft 365 offered by 21 Vianet, TLS 1.0/1.1 will be disabled on June 30, 2023.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md]:
```suggestion
For Microsoft 365 operated by 21 Vianet, TLS 1.0/1.1 will be disabled on June 30, 2023.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @qiufasheng, kindly commit the suggested change. Many thanks.

**Comment by qiufasheng** [comment]
@JohanFreelancer9, committed change based on your suggestion. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**